packages/outline-templates/default/tsconfig.json

```diff
@@ -42,6 +42,7 @@
   "exclude": [
     "./src/components/**/*.stories.ts",
     "./src/components/**/*.lit.ts",
+    "./src/components/sample/**/*",
```

Comment:
We exclude `src/components/examples/**` above in the `rollup` config, so should we also exclude them here?

Reply:
It might make sense to (or maybe examples is old and should just be removed). I'll see what others have to say to help guide the right end state.

Reply:
Do we want to ignore TS checks on samples and stories? I would think we want them regardless of their existence in the build to make sure our stories, examples, and samples are proper TS?

Reply:
This was added due to [this error](https://github.com/phase2/outline/actions/runs/4846050499/jobs/8635389374) though that error may be an artifact of the local linking to facilitate testing ability to initialize a project that builds successfully. Perhaps I should only inject this into the `tsconfig.json` and `rollup.config.mjs` file during the CI process?

Or, if there is a better adjustment you are aware of that you think might allow the us to keep the files included I'm all for it (for example, should I inject the linked outline directory via https://www.typescriptlang.org/tsconfig#rootDirs)?

Reply:
I think this issue might be irrelevant to what your trying to do here. I'm fine with merging this and dealing with it later. samples shouldnt really have typescript issues apart from missing or unused imports so I think its okay. I was just raising a general question.